content/webapp/pages/index.tsx

```diff
@@ -12,7 +12,6 @@ import { ArticleBasic } from '@weco/content/types/articles';
 import Space from '@weco/common/views/components/styled/Space';
 import Layout, { gridSize10 } from '@weco/common/views/components/Layout';
 import SimpleCardGrid from '@weco/content/components/SimpleCardGrid/SimpleCardGrid';
-import PageHeaderStandfirst from '@weco/common/views/components/PageHeaderStandfirst/PageHeaderStandfirst';
```

Comment:
Is it used anywhere else now? (should it be deleted)

Reply:
it is used in the Standfirst slice file (might be worth renaming it at some point)